x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint_list/endpoints.cy.ts

```diff
@@ -32,7 +32,7 @@ import { createEndpointHost } from '../../tasks/create_endpoint_host';
 import { deleteAllLoadedEndpointData } from '../../tasks/delete_all_endpoint_data';
 import { enableAllPolicyProtections } from '../../tasks/endpoint_policy';
 
-describe('Endpoints page', { tags: '@ess' }, () => {
+describe('Endpoints page', { tags: ['@ess', '@serverless', '@brokenInServerless'] }, () => {
```

Comment:
is there still work to be done to remove `@brokenInServerless` and fully enable these tests or is the intention to enable them in a followup?

Reply:
all tests require https://github.com/elastic/kibana/pull/167446, so if we merge this PR before Paul's PR then he could verify on his PR if roles are working properly in E2E